examples/create-react-app/src/tests/App.test.js

```diff
@@ -6,7 +6,7 @@ import App from '../App';
 import {mountWithAppProvider} from './utils/enzyme';
 
 // Sample test to demonstrate how you should test with AppProvider context
-it('renders page', () => {
+test('renders page', () => {
```

Comment:
I don't understand this change.

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md are the docs for this rule.

It seems we're using the default configuration of top level tests use `test` and all tests nested within `describe` use `it`.